.github/workflows/prune-dist.yml

```
@@ -22,20 +24,22 @@ jobs:
         run: |
           git clone https://github.com/${{ github.repository }}-dist.git dist-repo
 
-      - name: Truncate history of -dist repo
+      - name: Delete old tags
         run: |
           set -ex
-          cd dist-repo
-          # keep one week of builds
-          # oldest commit that applies to any still present tarball
-          REF=$(git log --before '7 days ago' --pretty=format:%H -1)
-          # empty if there are no old commits
-          [ -n "$REF" ] || exit 0
 
-          git checkout --orphan temp $REF
-          git commit -m "Truncated history"
-          git rebase --onto temp $REF main
-          git branch -D temp
-          git reflog expire --expire=now --all
+          HEAD=$(git rev-parse HEAD)
 
-          git push -f
+          cd dist-repo
+          now=$(date +%s)
```

my usual quoting complaints :)
```suggestion
          now="$(date +%s)"
```